extension/src/popup/components/sendPayment/SendSettings/index.tsx

```diff
@@ -105,22 +107,42 @@ export const SendSettings = ({
       ];
 
       const transaction = transfer(assetAddress, params, memo, builder);
-      const preflightSim = await sorobanClient.server.simulateTransaction(
-        transaction,
-      );
-
-      if ("transactionData" in preflightSim) {
-        dispatch(
-          saveSimulation({
-            response: preflightSim,
-            raw: transaction,
+      try {
+        const networkDetails = await getNetworkDetails();
+        const options = {
+          method: "POST",
+          headers: {
+            "Content-Type": "application/json",
+          },
+          body: JSON.stringify({
+            signex_xdr: transaction.toXDR(),
+            network_url: networkDetails.networkUrl,
+            network_passphrase: networkDetails.networkPassphrase,
           }),
+        };
+        const response = await fetch(`${INDEXER_URL}/simulate-tx`, options);
+        const preflightSim = await response.json();
+        console.log(preflightSim);
```

Comment:
console left in

Reply:
thanks, removed in 141c56475007f1899203f5255ccadde1a6ab9485